### DISCUSSION TITLE
Euro module minidexED DX7

### DISCUSSION THREAD

**Discussion by DutchBigEd** [probonopd/MiniDexed#478] [Show and tell]
I made an Euro module DX7 minidexED, using a Raspberry PI Zero WH so one instance only, but with a midi input.
![20230414_102815](https://user-images.githubusercontent.com/17274454/231996452-aa984471-74d1-40f9-a2d6-601f8758a71d.jpg)
![20230414_102821](https://user-images.githubusercontent.com/17274454/231996458-901940dd-f3a7-4ac1-b21a-ff0a25c93a9a.jpg)
![20230414_102829](https://user-images.githubusercontent.com/17274454/231996460-21fde508-01eb-4ee2-bb02-f5440d2af7f8.jpg)

**Comment by Banana71** [comment]
I like it very much!
it's a pity that it hasn't been able to buy an RPi Zero 2W for so long

How you write minidexED?  :-)

**Reply by DutchBigEd** [reply]
My name is ED ;)

**Comment by diyelectromusic** [comment]
That looks great!  Well done!

Are you now planning 7 more to make a Euro-Rack TX816?

Kevin

**Reply by DutchBigEd** [reply]
No, I have a Raspberry PI4b for that with 8 instances so I can run performances.

**Comment by DutchBigEd** [comment]
This is my implementation of the midi input.
![20230416_105059](https://user-images.githubusercontent.com/17274454/232287909-1c809b4c-27b0-4461-9681-5eca127a6945.jpg)
